src/zkp/piaffg.rs

```diff
@@ -134,67 +192,166 @@ impl Proof for PiAffgProof {
         transcript: &mut Transcript,
         rng: &mut R,
     ) -> Result<Self> {
-        // Sample alpha from 2^{ELL + EPSILON}
-        let alpha = random_plusminus_by_size(rng, ELL + EPSILON);
-        // Sample beta from 2^{ELL_PRIME + EPSILON}.
-        let beta = random_plusminus_by_size(rng, ELL_PRIME + EPSILON);
-
-        let (b, r) = input
-            .pk0
-            .encrypt(rng, &beta)
+        // The proof works as follows.
```

Comment:
This is great. I was a little wary about adding a different notation here, but I think it's a useful explanation and not that hard to map onto the more descriptive names in the code.